NorthstarDLL/mods/modmanager.cpp

```diff
@@ -267,6 +352,61 @@ ModManager::ModManager()
 	LoadMods();
 }
 
+struct Test
+{
+	std::string funcName;
+	ScriptContext context;
+};
+
+template <ScriptContext context> auto ModConCommandCallback_Internal(std::string name, const CCommand& command)
+{
+	if (g_pSquirrel<context>->m_pSQVM && g_pSquirrel<context>->m_pSQVM)
+	{
+		std::vector<std::string> args;
+		args.reserve(command.ArgC());
+		for (int i = 1; i < command.ArgC(); i++)
+			args.push_back(command.Arg(i));
+		g_pSquirrel<context>->AsyncCall(name, args);
+	}
+}
+
+auto ModConCommandCallback(const CCommand& command)
+{
+	ModConCommand* found = nullptr;
```

Comment:
generally more spacing between statements in this function would be nice, not many newlines